### PR TITLE
Compute SDK update - custom specs

### PR DESCRIPTION
**Pull request by ilij4**

### CODE REVIEW

**Review by rekpero — APPROVED** [review]
just check my RC on doc, otherwise it lgtm

File: packages/compute/src/instance-manager.ts [hunk 1]
```diff
@@ -39,28 +39,73 @@ class InstanceManager {
...
+      }
+    }
 
-    if (!computeMachine) {
-      throw new Error(
-        `Compute machine with id ${creationConfig.configuration.machineImageId} not found!`
+    if (
+      creationConfig.configuration.persistentStorage &&
+      (creationConfig.configuration.persistentStorage.size > 1024 ||
+        creationConfig.configuration.persistentStorage.size < 1)
```
Comment (shomix) [inline packages/compute/src/instance-manager.ts:74]: can it be 1?

File: packages/compute/src/instance-manager.ts [hunk 2]
```diff
@@ -144,6 +189,42 @@ class InstanceManager {
...
+      (createConfig.persistentStorage.size > 1024 ||
+        createConfig.persistentStorage.size < 1)
+    ) {
+      throw new Error(`Persistent storage must be number between 1 and 1024!`);
+    }
+
+    if (createConfig.storage > 1024 || createConfig.storage < 1) {
+      throw new Error(`Instance storage must be number between 1 and 1024!`);
+    }
+
```
Comment (shomix) [inline packages/compute/src/instance-manager.ts:227]: is the logic the same? Why don't we extract it in a fuction? Why don't we check the machine image id here?

File: packages/compute/src/instance-manager.ts (hunk 1 above)
Comment (ilij4) [reply packages/compute/src/instance-manager.ts]: @shomix yes, it is checking only if its less than 1

**Review by shomix — APPROVED** [review]

**Review by rekpero — APPROVED** [review]
Lgtm

**Pull request merged**